tests/ConstantTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Lambdish\Phunctional\Tests;
+
+use PHPUnit_Framework_TestCase;
+use function Lambdish\Phunctional\constant;
+
+final class ConstantTest extends PHPUnit_Framework_TestCase
+{
+    /** @test */
+    public function it_should_return_a_value()
+    {
+        $value = 5;
+        $constantClosure  = constant($value);
+
+        $this->assertSame($value, $constantClosure());
+    }
+
+    /** @test */
+    public function it_should_return_same_value_if_called_twice()
+    {
+        $value = 5;
+        $constantClosure  = constant($value);
+
+        $constantClosure();
+
+        $this->assertSame($value, $constantClosure());
```

Comment:
Do 2 times the `assertSame`

Reply:
Is really needed it? That test purpose to test is to test that when called more than one time it returns the same value. There's another test that test that when called once it returns the same value.